<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PowerGradeDlg</class>
 <widget class="QDialog" name="PowerGradeDlg">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>352</width>
    <height>394</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>352</width>
    <height>394</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>352</width>
    <height>394</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>370</width>
    <height>0</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Times New Roman</family>
    <pointsize>12</pointsize>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>电压等级定义</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../mainwindow.qrc">
    <normaloff>:/image/config.png</normaloff>:/image/config.png</iconset>
  </property>
  <widget class="QLabel" name="stationLabel">
   <property name="geometry">
    <rect>
     <x>24</x>
     <y>7</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>厂站选择</string>
   </property>
  </widget>
  <widget class="QComboBox" name="stationCombo">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>11</y>
     <width>231</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>47</y>
     <width>161</width>
     <height>321</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="title">
    <string>选择电压等级</string>
   </property>
   <widget class="QListWidget" name="selectList">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>18</y>
      <width>161</width>
      <height>301</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>47</y>
     <width>161</width>
     <height>321</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="title">
    <string>已选电压等级</string>
   </property>
   <widget class="QListWidget" name="selectedList">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>18</y>
      <width>161</width>
      <height>301</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QCheckBox" name="chineseCheckBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>中文显示 </string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../mainwindow.qrc"/>
 </resources>
 <connections/>
</ui>
